bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/robot/ProblemRobot.scala

```diff
@@ -161,6 +161,12 @@ class ProblemRobot(s: ShellAdvisor) extends Robot {
       msg = s"""Target "${t.name}" appears to be empty."""
     } yield new Problem(Severity.Error, msg, "Targets", s.inTargetsView(_.edit(t)))
 
+    private lazy val tooTargetsAndNoActivation = for {
+      t @ TooTarget(_, _) <- p.targets
+      if Proposal.toO(p.some).exists(_ == TooOption.None)
+      msg = "ToO targets not allowed. Please remove or set the TOO Activation"
```

Comment:
"ToO targets not allowed." seems a bit too definitive perhaps.  Maybe add "in non ToO proposals" or something.  We should also change TOO Activation to ToO Activation.  ToO is used in the [public documentation](http://www.gemini.edu/?q=node/11005).